udata_hydra/db/resource.py

```diff
@@ -0,0 +1,89 @@
+from udata_hydra import context
+from udata_hydra.utils.db import compute_update_query
+
+
+class Resource:
+    """Represents a resource in the "catalog" DB table"""
+
+    STATUSES = {
+        "TO_CHECK": "to be checked",
+        "TO_ANALYZE": "to be analyzed by CSV detective",
+        "TO_INSERT": "to be inserted in DB",
+        "CHECKED": "check finished",
+    }
```

Comment:
Not sure if it's worth using an [enum](https://docs.python.org/fr/3/library/enum.html) here?

Probably just an habit I got from using union types in elm :D

Reply:
I asked myself the exact same question., and I tried with an enum, but it made code more complicated to me, so I switched back to this which is simple and easily readable. Seems that enum in Python is not as simple as this. But if you think enum is really cleaner let me know!

Reply:
Nope, I haven't tried it in this case, so I'll trust you completely on keeping it as is, it was just a thought, thanks for explaining the reasoning behind it 🙏